jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/viewmodel/RecordableViewModel.kt

```diff
@@ -318,26 +291,46 @@ open class RecordableViewModel(
             .doOnError { e ->
                 logger.error("Error in removing deleted take: $take", e)
             }
+            .observeOnFx()
             .subscribe {
-                removeFromAlternateTakes(take)
+                val isTakeSelected = takeCardModels.any { it.take == take && it.selected }
+                if (isTakeSelected) {
+                    takeCardModels.firstOrNull()?.let {
+                        selectTake(it.take)
+                    }
```

Comment:
I think we may want the most recent take in the list (the last one) instead of the first take.